app/Http/Controllers/User/CouponUserController.php

```diff
@@ -32,6 +34,8 @@ public function applyAccessCode($code_hash)
                         $this->incrementCodeUses($code_hash);
                         $this->updateCodesUsesTable($code_hash, $pid);
                         $code['first_aid'] = CourseSyllabus::where('course_id', $cid)->where('seq', 0)->first()->activity_id;
+
+                        auth()->user()->notify(new AccessGrantedNotification($cid));
```

Comment:
Hmmm... is this correct? I remember us both wondering why the notification is going through user object.

Reply:
ah no it's not but that code was added months ago so don't blame me